bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -232,26 +168,22 @@ <h3>Private Relay</h3>
     <li>relay.firefox.com</li>
   </ul>
 
-  <h3>Ship It</h3>
-  <ul class="mzp-u-list-styled">
-    <li>shipit-api.mozilla-releng.net</li>
-    <li>shipit.mozilla-releng.net</li>
-  </ul>
-
   <h3>Source Control</h3>
   <ul class="mzp-u-list-styled">
     <li>hg.mozilla.org (website only)</li>
   </ul>
 
-  <h3>Speak To Me</h3>
+  <h3>VPN</h3>
   <ul class="mzp-u-list-styled">
-    <li>speaktome-2.services.mozilla.com</li>
-    <li>speaktome.services.mozilla.com</li>
+    <li>vpn.mozilla.org (VPN client)</li>
```

Comment:
@dveditz do we remove the VPN client from the web bounty list and mention it in the client or keep it as-id?

Reply:
"vpn.mozilla.org" is an internet service _used by_ the client, but it's not actually the client. We definitely want the bounty to cover the server if something's wrong. Maybe just remove the paranthetical?